packages/apps/docs/src/docs/build/index.tsx

```diff
@@ -43,9 +44,12 @@ const Home: FC<IProps> = ({ blogPosts, popularPages }) => {
                 clicking Quick start.
               </Text>
             </Box>
-            <Button as="a" asChild icon="TrailingIcon">
-              <Link href={'/build/quickstart'}>Quick start</Link>
-            </Button>
+            <Link
```

Comment:
here the styling is gone. I understand it is a link. but it had the styling of a button